<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>562</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="GetStarted">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>50</y>
      <width>231</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Poppins SemiBold</family>
      <pointsize>26</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Get Started</string>
    </property>
   </widget>
   <widget class="QLabel" name="Examples_1">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>260</y>
      <width>191</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Poppins Light</family>
      <pointsize>12</pointsize>
      <underline>true</underline>
     </font>
    </property>
    <property name="text">
     <string>Basic gNB &amp; Core</string>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>110</y>
      <width>261</width>
      <height>140</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="NewTopo">
       <property name="font">
        <font>
         <family>Poppins Medium</family>
         <pointsize>18</pointsize>
         <underline>true</underline>
        </font>
       </property>
       <property name="text">
        <string>New Topo</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="OpenTopo">
       <property name="font">
        <font>
         <family>Poppins Medium</family>
         <pointsize>18</pointsize>
         <underline>true</underline>
         <kerning>true</kerning>
        </font>
       </property>
       <property name="text">
        <string>Open Topo</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="Examples">
       <property name="font">
        <font>
         <family>Poppins Medium</family>
         <pointsize>18</pointsize>
         <underline>false</underline>
        </font>
       </property>
       <property name="text">
        <string>Examples</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="NetFluxLogo">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>60</y>
      <width>371</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../Icon/logo netflux.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="Description">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>160</y>
      <width>351</width>
      <height>121</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Poppins Light</family>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>NetFlux 5G is a visual simulation and planning tool for 5G network topologies. Designed for researchers, students, and network professionals, it allows you to build, configure, and emulate 5G infrastructures. NetFlux 5G offers a flexible and powerful platform to bring your designs to life.</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>270</y>
      <width>291</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Poppins Medium</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Start building your 5G network now!</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="ExamplesIcon_1">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>260</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../Icon/folder.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="Examples_2">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>290</y>
      <width>191</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Poppins Light</family>
      <pointsize>12</pointsize>
      <underline>true</underline>
     </font>
    </property>
    <property name="text">
     <string>Multi Core Deployment</string>
    </property>
   </widget>
   <widget class="QLabel" name="ExamplesIcon_2">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>290</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../Icon/folder.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="Examples_3">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>320</y>
      <width>191</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Poppins Light</family>
      <pointsize>12</pointsize>
      <underline>true</underline>
     </font>
    </property>
    <property name="text">
     <string>SDN Deployment</string>
    </property>
   </widget>
   <widget class="QLabel" name="ExamplesIcon_3">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>320</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../Icon/folder.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
